services/retention/src/main/java/app/coronawarn/datadonation/services/retention/runner/RetentionPolicy.java

```diff
@@ -175,6 +178,13 @@ private void deleteOutdatedSalt() {
     saltRepository.deleteOlderThan(saltThreshold);
   }
 
+  private void deleteOutdatedScanInstance() {
+    final LocalDate date = threshold(retentionConfiguration.getExposureWindowRetentionDays());
```

Comment:
I assume exposure window retention and scan instance retention is the same?

Reply:
yes